<launch>
	<node pkg="task2" type="toroid_detector3.py" name="toroid_detector" output="screen" />
	<node pkg="task2" type="terminal_approach.py" name="terminal_approach" output="screen" />
	<node pkg="task2" type="main_test.py" name="main" output="screen" />
	<node pkg="task2" type="feature_builder.py" name="feature_builder" output="screen" />
	<node pkg="task2" type="sound_player.py" name="sound_player" output="screen" />
	<!-- <node pkg="task2" type="music_node.py" name="music_node" output="screen" /> -->
	<node pkg="sound_play" type="soundplay_node.py" name="soundplay_node" output="screen" />
</launch>
